collection/lscq/asm_arm64.s

```diff
@@ -0,0 +1,89 @@
+#include "textflag.h"
+#include "funcdata.h"
+
+TEXT ·compareAndSwapUint128(SB), NOSPLIT, $0-41
```

Comment:
I think we should use `$0-48`, the function on arm64 is 64-bit aligned, so a `bool` will use 64-bit space instead one byte.

Reply:
Sorry for the delay, I'm occupied away from github recently.
Yes, the 64bit aligned makes sense. But I've just checked the go src:
in `$GOROOT/src/runtime/internal/atomic/atomic_arm64.s`, the frame size of `Cas64(uint64 *ptr, uint64 old, uint64 new) bool` is `$0-25`:
```go
TEXT ·Cas64(SB), NOSPLIT, $0-25
	MOVD	ptr+0(FP), R0
	MOVD	old+8(FP), R1
	MOVD	new+16(FP), R2
	MOVBU	internal∕cpu·ARM64+const_offsetARM64HasATOMICS(SB), R4
	CBZ 	R4, load_store_loop
	MOVD	R1, R3
```
Maybe the go compiler do it for us.

Reply:
Sounds good! Keep sync with the Go runtime is better.